codelabs/custom_renderer/custom_renderer.md

```diff
@@ -18,7 +18,7 @@ Feedback Link: https://github.com/google/blockly-samples/issues/new/choose
 
 ### What you'll build
 
-You will build and use four renderers:
+This codelab build and use four renderers:
```

Comment:
```suggestion
This codelab builds and uses four renderers:
```

I'm not actually sure if this is correct :P

Reply:
yeah depends on how you look at it :) i think it's arguably correct!

Reply:
should be fixed now.  thanks!